### PR TITLE
introduce ecosystem page of users and service providers

### PR DESCRIPTION
**Pull request by DanielJonesEB**
The organisations on this list were identified either by:

* appearance on the old [Concourse in the Wild wiki page][0]
* direct approach via email/Slack
* existence of comments, presentations, posts in the public domain

[0]: https://github.com/concourse/concourse/wiki/Concourse-in-the-Wild

### CODE REVIEW

**Comment by DanielJonesEB** [comment]
The intention here is to provide social proof of Concourse's usage, for the more fashion-oriented technology decision-maker. We know that there are over a hundred organisations using Concourse, and this aims to be a bootstrapping list.

I think it's important to also allow people the opportunity to market their Concourse services. In order to ensure that the Concourse project remains healthy, we need people with a commercial interest in maintaining and extending it. This is a small step towards bringing together supply and demand, whilst also instilling confidence in potential users that they're not on their own.

File: lit/ecosystem.lit [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+its user base, and include CI/CD for apps, continuous delivery of
+infrastructure, release integration, automation of tests, and many more!
+
+If you use Concourse, or your business offers Concourse-related services, we'd
+love to hear from you. Please \link{raise a pull
+request}{https://github.com/concourse/docs/pulls} adding your organisation's
+name in lexicographical order to the list below, and help us let the world know
+how many people do things continuously with Concourse.
+
+\section{
```
[the remark this answers is not in the document]
Comment (taylorsilva) [reply lit/ecosystem.lit:16]: This is how we would have done it too.

**Comment by taylorsilva** [comment]
Thanks for the PR Daniel! This is useful to showcase especially for new users 👍 
Do you have a few more people you're waiting on emails from that you'd like to eventually add to this PR?

**Comment by DanielJonesEB** [comment]
> Thanks for the PR Daniel! This is useful to showcase especially for new users 👍

No worries, I should have done this years ago!

> Do you have a few more people you're waiting on emails from that you'd like to eventually add to this PR?

I do, but I think they might take a while to get back to me, so I think it'd be best to raise new PRs for those folks. As soon as this is merged and published, we can point people to it on Discord/Discussion/where-ever and ask them to add themselves.

We've got a list of >100 Concourse users, including some very big brands, but sadly we don't have the details of the right people to approach. So if we can get them to self-elect, all the better :)

**Review by taylorsilva — APPROVED** [review]
Approving, thanks again Daniel! This won't be published until the next release, but that should happen in the next ~2 weeks. I'm hoping to make some changes to this repo so stuff like this can be published immediately instead of stuck waiting for the next release.

**Pull request merged**